src/components/Book/Book.js

```diff
@@ -1,16 +1,21 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchData } from './src/Redux/books/books.js';
+import Books from './Books';
 
-const Book = ({ title, author }) => (
-  <>
-    <h2 className="title">{title}</h2>
-    <p className="author">{author}</p>
-  </>
-);
+const Book = () => {
+  const books = useSelector((state) => state.booksReducer);
+  const dispatch = useDispatch();
 
-Book.propTypes = {
-  title: PropTypes.string.isRequired,
-  author: PropTypes.string.isRequired,
+  useEffect(() => {
+    dispatch(fetchData());
+  }, []);
+
+  return (
+    <div>
+      {books.map((book) => <Books key={book.item_id} book={book} />)}
```

Comment:
Books get disappeared when I refresh the page. Please make sure that you display books on every re-render of the component.

Reply:
Thanks @SajjadAhmad14 I have Implemented